README.md

```diff
@@ -1,24 +1,38 @@
 # Quick Setup Locally
 
 > Install the Dependencies
+
 ```
 cd daily-code
 yarn install
 ```
+
 > Copy the env example
+
 ```
 cd /packages/db
 cp .env.example .env
 ```
->Update the .env file with the database url
+
+> Update the .env file with the database url
 
 > Migrate and the Database
+
 ```
 npx prisma migrate dev
 npx prisma db seed
 ```
+
+> If previous commands fail, try these; Otherwise, skip.
+
+```
+yarn prisma migrate dev
```

Comment:
Is this right?
Or do we need to add more things form here - https://github.com/code100x/daily-code/issues/196#issuecomment-2052235001?

Reply:
Yes, it works in my branch.
<img width="1010" alt="Screenshot 2024-04-13 at 8 29 07 PM" src="https://github.com/code100x/daily-code/assets/58658277/8882d044-6345-48bd-a4c8-a7ebd2eb1440">
But, I think we should add the npx prisma generate in the middle as well because you also faced the seed failing issue and had to regenerate the client.
I'll add that commit too if it is needed.